src/cli.ts

```diff
@@ -3,19 +3,66 @@ import { existsSync } from 'node:fs';
 
 const args = () => process.argv.slice(2);
 
+const getFlag = (flag: string): string | undefined => {
+  const flagIndex = args().indexOf(flag);
+  if (flagIndex !== -1 && args()[flagIndex + 1]) {
+    return args()[flagIndex + 1];
+  }
+  return undefined;
+};
+
```

Comment:
`undefined` is the default return so you can exclude that. Overall this function can be simplified to: 
```suggestion
const getFlag = (flag: string): string | undefined => {
  const flagIndex = args().indexOf(flag);
  return flagIndex !== -1 ? args()[flagIndex + 1] : undefined;
};
```